packages/office-ui-fabric-react/src/components/ColorPicker/ColorRectangle/ColorRectangle.base.tsx

```diff
@@ -70,7 +70,9 @@ export class ColorRectangleBase extends BaseComponent<IColorRectangleProps, ICol
         <div
           className={classNames.thumb}
           style={{ left: color!.s + '%', top: MAX_COLOR_VALUE - color!.v + '%', backgroundColor: color!.str }}
-        />
+        >
+          <div className={classNames.thumbInner} />
```

Comment:
Hmm. Do we really need another div here? Why not use a pseudo element? Or inner box-shadow? There are so many options.

DOM is expensive so any opportunities to reduce it is good.

Reply:
Right. It already has box-shadow, so I'll go with a pseudo element.